grml-debootstrap

```diff
@@ -75,7 +75,7 @@ MNTPOINT="/mnt/debootstrap.$$"
 [ -n "$TIMEZONE" ] || TIMEZONE='Europe/Vienna'
 [ -n "$TUNE2FS" ] || TUNE2FS='tune2fs -c0 -i0'
 [ -n "$UPGRADE_SYSTEM" ] || UPGRADE_SYSTEM='yes'
-[ -n "$VMSIZE" ] || VMSIZE="2G"
+#[ -n "$IMAGESIZE" ] || IMAGESIZE="2G"
```

Comment:
?

Reply:
Was shuffling code around and left "old code" commented out to avoid making destructive changes mid-work. I should remove that line before taking this out of draft status.

Reply:
Removed.